css/css-flexbox/flexbox_first-letter.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<title>flexbox | first-letter</title>
+
```

Comment:
Please add a meta tag with an assertion of what this test is trying accomplish eg: `<meta name="assert" content="This test is ensuring that the flexbox properties do not work on the :first-letter pseudo element" />`. While this can feel redundant - there are times where someone does not test what they set out to test. Putting it in a sentence allows someone reviewing it later to know what you were trying to test. Additionally, the spec may have changed - and your assertion may point to the fact that the test is outdated and no longer relevant.

Reply:
Noted 👍